raiden-dapp/src/components/notification-panel/NotificationCard.vue

```diff
@@ -74,6 +82,16 @@ export default class NotificationCard extends Vue {
       return false;
     }
   }
+
+  get linkRoute() {
+    switch (this.notification.dappRoute) {
+      case 'backup state route':
+        return () => this.navigateToBackupState();
+        break;
+      default:
+        break;
+    }
+  }
```

Comment:
Can I suggest you a better solution? 
I really like the Open-Closed-Principle. The idea is that the notification card is able to handle optional links of a notification. But you don't want to adjust the component each time a new link got added. It is cumbersome to have the developer remind this and you simply clutter the logic of the notification card component.
I would rather replace the `notification.link` with a `RouteName` (use the typing). And then just have a function that uses `this.$router.push(name: this.notification.link)` (or similar). Thereby you have written the logic once. As long as the idea of the functionality remains the same this remains "closed". But it is open for extension, because each notification can simply define this link name and it works out of the box. When you add a new notification or extend an existing one with a link you don't have to remember adding a switch-case here. You can get rid of the Mixin (which bloats up the final package/user load time) and due to the enumeration of the route names you are also type safe (note to then not define the links in the locals, but where you dispatch the configuration).
Sorry for the long comment and the critics. I hope you understand my arguments.

Reply:
This is much better, thank you for the suggestions.